build.gradle.kts

```diff
@@ -45,9 +45,9 @@ extensions.configure<ExtraPropertiesExtension>("ext") {
 gradlePlugin {
     plugins.create("conventionsPlugin") {
         id = "io.github.gradle.gradle-enterprise-conventions-plugin"
-        implementationClass = "com.gradle.enterprise.conventions.GradleEnterpriseConventionsPlugin"
-        displayName = "Gradle Enterprise Conventions Plugin"
-        description = "Gradle Enterprise Conventions Plugin"
+        implementationClass = "com.gradle.enterprise.conventions.DevelocityConventionsPlugin"
+        displayName = "Develocity Conventions Plugin"
```

Comment:
Is this published to the plugin portal? Then we should make clear that this is a Develocity conventions plugin for the Gradle Github organization.

Reply:
Yes it is: https://plugins.gradle.org/plugin/io.github.gradle.gradle-enterprise-conventions-plugin

Reply:
Again, what is the benefit of a separate PR?